### ISSUE TITLE
Need better workmode onboarding

### ISSUE THREAD

**Issue by cwilso** [whatwg/sg#214]
(From suggestion by Mu-An at WHATUP)

We should ensure that feature drivers have clear pointers to the workmode, and then when/if we add the Spec Acceptance Stages that we inform feature developers of this new tool to help manage expectations.

**Comment by cwilso** [comment]
"The PR template is nice, but we should have an issue template that points to the Working Mode"

**Comment by domenic** [comment]
@muan

**Comment by annevk** [comment]
I agree we should do this, but it probably makes sense to track this over in whatwg/meta instead of here? Or do you think the SG should be involved in some way?

What I'd like to do is patch https://github.com/whatwg/spec-factory so it can push this out everywhere as well as better READMEs.

**Comment by cwilso** [comment]
No, you're probably right; I was making sure I wouldn't forget about it.  I'll move it to meta.

**Comment by cwilso** [comment]
Oh: @annevk you'll have to transfer it, or give me admin rights on meta.
